_ingest-pipelines/processors/ml-inference.md

```diff
@@ -0,0 +1,158 @@
+| `max_prediction_tasks` | Int | Optional | The setting of the maximum number of prediction tasks can run when ingesting a document, default value is 10.                                                                                                                                                                                                                                                                                                     |
+| `description` | String | Optional | A brief description of the processor.                                                                                                                                                                                                                                                                                                                                                                             |
+| `tag` | String | Optional | An identifier tag for the processor. Useful for debugging to distinguish between processors of the same type.                                                                                                                                                                                                                                                                                                     |
+| `ignore_missing` | Boolean | Optional | If true and any of the input fields defined in input_map or output_map are missing, those missing fields are quietly ignored. Otherwise, a missing field causes a failure. Default is false.                                                                                                                                                                                                                      |
+| `ignore_failure` | Boolean | Optional | If true and any exceptions occur, they are quietly ignored, and the ingestion continues. Otherwise, an exception can occur and abort the ingestion. Default is false.                                                                                                                                                                                                                                             |
+
+## Using the processor
+
+Follow these steps to use the processor in a pipeline. You must provide a model ID when creating the processor. For more information, see [Using custom models within OpenSearch]({{site.url}}{{site.baseurl}}/ml-commons-plugin/using-ml-models/).
+
+**Step 1: Create a pipeline.**
+
+The following example request creates an ingest pipeline where the model requires an `input` field and produces a `data` field in the prediction task and the document has a `passage_text` field, which  will be converted into text embeddings and the embeddings will be stored in `passage_embedding` field. 
+
+The mappings for input_map and output_map also support standard json path notation to complex data structures.
```

Comment:
How about adding link to "standard json path", so user can learn more details

Reply:
add link to https://docs.oracle.com/cd/E60058_01/PDF/8.0.8.x/8.0.8.0.0/PMF_HTML/JsonPath_Expressions.htm